bin/ignite.bat

```diff
@@ -119,8 +118,10 @@ if %ERRORLEVEL% neq 0 (
 ::
 :: Process 'restart'.
 ::
-set RANDOM_NUMBER_COMMAND="!JAVA_HOME!\bin\java.exe" -cp %CP% org.apache.ignite.startup.cmdline.CommandLineRandomNumberGenerator
-for /f "usebackq tokens=*" %%i in (`!RANDOM_NUMBER_COMMAND!`) do set RANDOM_NUMBER=%%i
+set RANDOM_NUMBER_COMMAND="!JAVA_HOME!\bin\java.exe" -cp "%CP%" org.apache.ignite.startup.cmdline.CommandLineRandomNumberGenerator
+for /f "usebackq tokens=*" %%i in (`"!RANDOM_NUMBER_COMMAND!"`) do (
+    set RANDOM_NUMBER=%%i
```

Comment:
Round brackets wrapping is not related to task

Reply:
I added it for better readability. Should I create a new task for this?

Reply:
Yes. We must stick to task only, no concomitant changes should be introduced that do not relate the task at least indirectly.